.github/workflows/i18n.yml

```
@@ -0,0 +1,32 @@
+name: i18n
+
+on:
+  # allow running manually
+  workflow_dispatch:
+  # run weekly
+  schedule:
+    - cron:  '30 5 * * 0'
+
+jobs:
+  i18n:
+    runs-on: ubuntu-latest
+    steps:
+
+    - uses: actions/checkout@v2
+    - name: "extract strings"
+      run: |
+        npm install
+        npm run gettext:extract
+
+    - name: "compile strings"
+      run: npm run gettext:compile
+
+    - name: "commit"
+      run: |
+        git config --global user.name 'GH Actions'
+        git config --global user.email 'gh_actions@users.noreply.github.com'
+        git remote set-url origin https://x-access-token:${{ secrets.GITHUB_TOKEN }}@github.com/ansible/ansible-hub-ui.git
+        git add locale/
+        if git commit -m "Automated updated of i18n strings on $(date +'%Y-%m-%d')"; then
+          git push --set-upstream origin
+        if
```

Ah :)

```suggestion
        fi
```

https://github.com/ansible/ansible-hub-ui/runs/4572955455?check_suite_focus=true#step:5:10

---

(fixed in https://github.com/ansible/ansible-hub-ui/pull/1460)